### PR TITLE
Add attaches to section feedback by CodeEditorField

### LINKED ISSUE #92: Add attaches to section feedback by CodeEditorField
we do not need DropScreenshot anymore

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #92

### CODE REVIEW

File: src/backend/modules/s3/s3-module.ts [hunk 1]
```diff
@@ -19,19 +19,24 @@ export const client = new S3Client({
...
+                ACL: 'authenticated-read',
+                ContentType: contentType,
+                CacheControl: 'no-cache',
+            }),
+        );
+
+        return;
+    } catch (error) {
+        // @ts-ignore
+        return error.message as string;
```
Comment (tsumo) [inline src/backend/modules/s3/s3-module.ts:38]: `error instanceof Error ? error.message : 'Upload error'`

Comment (tsumo) [inline src/backend/modules/s3/s3-module.ts:38]: How to distinguish success from error? They both are string

File: src/api/attach/attach-hooks.ts [hunk 2]
```diff
@@ -11,17 +11,23 @@ type UploadFiles = {
     formData: FormData;
 };
 
-export const useAttachesCreateMutation = (): UseMutationResult<Attach, unknown, UploadFiles> => {
+export const onUploadSuccess = (sectionId: number) => {
```
Comment (tsumo) [inline src/api/attach/attach-hooks.ts:14]: Technically it is a hook, but it's name does not start with "use". I think onSuccess and onFail should be useCallback functions on the section feedback page

**Review by tsumo — CHANGES_REQUESTED** [review]

**Comment by tsumo** [comment]
Rebase is needed, because of the merged linters

File: src/backend/modules/s3/s3-module.ts (hunk 1 above)
Comment (nyet-ty) [reply src/backend/modules/s3/s3-module.ts:38]: success will return undefined, after I'll correct catch to `error instanceof Error ? error.message : 'Upload error'` fail will be always string

**Review by tsumo — APPROVED** [review]

**Pull request merged**